resources/public/api/conf/common/testing.md

```diff
@@ -1,24 +1,13 @@
-You can use the sandbox environment to <a href="/api-documentation/docs/testing">test this API</a>. You can use the <a href="/api-documentation/docs/api/service/api-platform-test-user/1.0">Create Test User API</a> or it's frontend service to create test users.
 
-It may not be possible to test all scenarios in the sandbox. You can test some scenarios by passing the Gov-Test-Scenario header. Documentation for each endpoint includes a **Test data** section, which explains the scenarios that you can simulate using the Gov-Test-Scenario header.
+Some APIs may be marked [test only](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing). 
+This means that they are not available for use in production and may change.
 
-For versions 2.0 and later, Gov-Test-Scenario values that are not supported by the sandbox will result in a status code 400 response with an error code of RULE_INCORRECT_GOV_TEST_SCENARIO.
-
-If you have a specific testing need that is not supported in the sandbox, contact <a href="/developer/support">our support team</a>.
-
-Some APIs may be marked \[test only\]. This means that they are not available for use in production and may change.
-
-### Stateful
-Some endpoints support STATEFUL gov test scenarios. Stateful scenarios work with groups of endpoints that represent particular types of submissions.  For each type you can POST (or PUT) to submit or amend data, GET to retrieve or list data and DELETE to delete data. For example, with a STATEFUL gov test scenario a retrieval will return data based on what you submitted.
-
-The following groups are stateful in the sandbox:
-- UK Property Business Annual Submission
-- Foreign Property Annual Submission
-- UK Property Income and Expenses Period Summary 
-- Foreign Property Income and Expenses Period Summary
-- Historic FHL UK Property Business Annual Submission
-- Historic FHL UK Property Income and Expenses Period Summary
-- Historic non-FHL UK Property Business Annual Submission
-- Historic non-FHL UK Property Income and Expenses Period Summary
+You can use the Sandbox environment to test this API. You can test different scenarios in the Sandbox by passing the
```

Comment:
'test this API' should be a link pointing to https://developer.service.hmrc.gov.uk/api-documentation/docs/testing